drupal_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -x
```

Comment:
`set -x` should probably be removed, to make the Travis output more readable (we could still display some specific messages at some points if we need them to understand what gets output)